screenpy/actions/see_all_of.py

```diff
@@ -37,26 +40,30 @@ class SeeAllOf:
         )
     """
 
+    tests: Tuple[T_T, ...]
+
     @classmethod
-    def the(cls, *tests: Tuple[Answerable, BaseResolution]) -> "SeeAllOf":
+    def the(cls: Type[SelfSeeAllOf], *tests: T_T) -> SelfSeeAllOf:
         """Supply any number of Question/value + Resolution tuples to test."""
         return cls(*tests)
 
-    def describe(self) -> str:
+    def describe(self: SelfSeeAllOf) -> str:
         """Describe the Action in present tense."""
         return f"See if all of {self.number_of_tests} tests pass."
 
     @beat("{} sees if all of the following {number_of_tests} tests pass:")
```

Comment:
Hm... now that we support a single test, we might need to be smarter about this description.

Reply:
"Actor sees if all of the following 1 test(s) pass:"  <-- would that work?
Anything smarter will require some logic that would be a bit more difficult to deal with in the beat decorator.

Reply:
I opened an issue for this #33 because it potentially starts to open up a can of worms surrounding how beat logging works.